### PR TITLE
Focus content on scroll out #108

### PR DESCRIPTION
**Pull request by Gozala**
Fix for #108

### CODE REVIEW

File: src/browser/browser.js [hunk 1]
```diff
@@ -132,6 +132,13 @@ define((require, exports, module) => {
 
     const theme = readTheme(activeWebViewerCursor);
 
+    const onNavigationPanelVisible = () => {
```
Comment (gordonbrander) [inline src/browser/browser.js]: Noop?

**Comment by paulrouget** [comment]
I'm not sure this will work well with apzc.

**Comment by Gozala** [comment]
> I'm not sure this will work well with apzc.

@paulrouget Can we deal with that if it / when it won't ? Also I could not think of any better way dinging it, if you have a suggestion I'm all ears.

**Comment by paulrouget** [comment]
> @paulrouget Can we deal with that if it / when it won't ? Also I could not think of any better way dinging it, if you have a suggestion I'm all ears.

It will work with APZC. But see Gordon comment in #108.

**Comment by Gozala** [comment]
We wend with #147 instead

**Pull request closed without merging**